### ISSUE TITLE
Allow credentials from .netrc

### ISSUE THREAD

**Issue by chrisbouchard** [satodoc/httpie-oauth2-client-credentials#2]
HTTPie allows plugins to opt-in to loading credentials from `.netrc`. (httpie/httpie#718) by setting `netrc_parse = True` in the plugin. I'd like to request that this plugin enable that feature, so that we can manage API credentials per-domain. I'd be happy to make a merge request if desirable.

**Comment by satodoc** [comment]
The target host that httpie retrieves from .netrc with get_auth is the TARGET_ENDPOINT host, and we did not intend to support `.netrc` on a different host than the one accessed to retrieve the token.

However, we have changed the README to support it with a note in "[Important Notes before Use](https://github.com/satodoc/httpie-oauth2-client-credentials#important-notes-before-use)" of the README.

Currently there is an [Issue#1449](https://github.com/httpie/httpie/issues/1499) problem with httpie.
I will confirm that the [PR#1501](https://github.com/httpie/httpie/pull/1501) that resolves this issue has been merged and the package has been published.
And I will then release this plugin package.
